woocommerce/woocommerce
# wc-admin: Include package

### All Submissions:






### Changes proposed in this Pull Request:



Add wc-admin as a dependency

### How to test the changes in this Pull Request:

1. Checkout this branch and run `composer update`
2. Check that wc-admin is present and working
3. Check version for conflicts

### Other information:




### Changelog entry

> Enhancement - New WooCommerce Admin.
